app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,45 @@
+class Api::V1::ReviewsController < ApiController
+
+  def index
+    render json: Review.all
+  end
```

Comment:
Do you need this, or are you serving it up via your `SuperpowersController#show`?